### DISCUSSION TITLE
How to remove arrow button on the scroll bar ?

### DISCUSSION THREAD

**Discussion by OriLiMu** [ScratchAddons/ScratchAddons#7652] [Help] [answered]
This code works but not work in my project.
```html
<!doctype html>
<html>
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style>
      body {
        margin: 0;
        padding: 0;
        height: 100%;
        overflow: auto;
      }
      div {
        height: 200vh;
        background-color: blue;
      }
      /* width */
      ::-webkit-scrollbar {
        width: 10px;
      }

      /* Handle */
      ::-webkit-scrollbar-thumb {
        background: #888;
      }
    </style>
  </head>

  <body>
    <div></div>
  </body>
</html>
```

I do some google  work .  Add the css script below but not work. Get stuck here.
```css
#floatWindow::-webkit-scrollbar-button {
  display: none; 
}
```
My project here. Not the lastest code. I make some change after this which cant check this issue.  Location: find-references folder in addons. 
https://github.com/woshichuanqilz/ScratchAddons/tree/1340f8fb2c6b3a04417501d92019c6f52761849d

![img](https://media.discordapp.net/attachments/1263048343529979975/1263048344532422676/image.png?ex=6698d0ef&is=66977f6f&hm=3cb0117fc94edbb216bb95bf6fc7fe0db504ff7f499fa29c23f2177fe017afa4&=&format=webp&quality=lossless&width=1650&height=1337)

**Comment by WorldLanguages** [comment]
I wouldn't be so sure that arrow is the browser scrollbar. Maybe it's an HTML element or a :before/:after pseudo-element? You should confirm what the arrow is (with the browser developer tools) first.

**Comment by mxmou** [comment, marked as answer]
The standard `scrollbar-width` and `scrollbar-color` properties override any styles applied to `::-webkit-scrollbar-*` pseudo-elements. The only way to remove the arrow buttons that works on all browsers is to make a custom scrollbar using JS, but I don't recommend doing that unless you have a specific reason.
